metricbeat/module/windows/perfmon/pdh_windows.go

```diff
@@ -77,6 +80,26 @@ func PdhGetFormattedCounterValue(counter PdhCounterHandle, format PdhCounterForm
 	return counterType, &value, nil
 }
 
+func PdhGetFormattedCounterArray(counter PdhCounterHandle, format PdhCounterFormat) (int, *[]PdhCounterValueItem, error) {
```

Comment:
After looking at the `PdhCounterValueItem` and how it's used, I think it would be best to return a different struct that has the a `Name` field that is a `string`. This way the users of the method don't need to follow the pointer and convert the UTF16 bytes to UTF8.